addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -1442,6 +1508,20 @@ export function fillEmpty(el) {
     }
     return fillers;
 }
+/**
+ * Takes a selection (assumed to be collapsed) and insert a zero-width space at
+ * its anchor point. Then, select that zero-width space.
+ *
+ * @param {Selection} selection
+ * @returns {Node} the inserted zero-width space
+ */
+export function insertAndSelectZws(selection) {
+    const offset = selection.anchorOffset;
+    const zws = insertText(selection, '\u200B');
```

Comment:
Shouldn't this use the fillEmpty mechanism instead ? Otherwise the zws won't be cleaned up on save.

Reply:
The problem is `fillEmpty` is well _filling_ something. This is just inserting it, possibly in the middle of some text.